shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsConfig.java

```diff
@@ -30,13 +30,18 @@
     public final static Property<String> CSV_ENDPOINT = Property.named("csvEndpoint", "/tmp/csv");
     public final static Property<String> STATSD_HOST = Property.named("statsDHost", "localhost");
     public final static Property<Integer> STATSD_PORT = Property.named("statsDPort", 8125);
+    public final static Property<String> INFLUXDB_URI = Property.named("influxDBURI", "http://localhost:8086");
+    public final static Property<String> INFLUXDB_NAME = Property.named("influxDBName", "pravega");
+    public final static Property<String> INFLUXDB_USERNAME = Property.named("influxDBUserName", "");
+    public final static Property<String> INFLUXDB_PASSWORD = Property.named("influxDBPassword", "");
     public final static Property<String> GRAPHITE_HOST = Property.named("graphiteHost", "localhost");
     public final static Property<Integer> GRAPHITE_PORT = Property.named("graphitePort", 2003);
     public final static Property<String> JMX_DOMAIN = Property.named("jmxDomain", "domain");
     public final static Property<String> GANGLIA_HOST = Property.named("gangliaHost", "localhost");
     public final static Property<Integer> GANGLIA_PORT = Property.named("gangliaPort", 8649);
     public final static Property<Boolean> ENABLE_CSV_REPORTER = Property.named("enableCSVReporter", true);
     public final static Property<Boolean> ENABLE_STATSD_REPORTER = Property.named("enableStatsdReporter", true);
+    public final static Property<Boolean> ENABLE_INFLUXDB_REPORTER = Property.named("enableInfluxDBReporter", false);
     public final static Property<Boolean> ENABLE_GRAPHITE_REPORTER = Property.named("enableGraphiteReporter", false);
     public final static Property<Boolean> ENABLE_JMX_REPORTER = Property.named("enableJMXReporter", false);
     public final static Property<Boolean> ENABLE_GANGLIA_REPORTER = Property.named("enableGangliaReporter", false);
```

Comment:
Haven't we just removed all these other ones? Let's get rid of them here too.

Also find anywhere else in the code these might still be showing up and clean them up too.

Reply:
Fixed